### PR TITLE
Fix issue #61 add invoke-chocolatey which throws error on choco error

### PR DESCRIPTION
**Pull request by esiebes**
https://github.com/chocolatey/cChoco/issues/61
resolution for issue 61. In our company me an my colleagues also noticed that cchoco always returned succes even when chocolatey gave an error. This fix wil throw an error if chocolatey gives an error.

### CODE REVIEW

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 [hunk 1]
```diff
@@ -419,11 +419,60 @@ function Get-ChocoInstalledPackage ($action) {
...
+        {
+            $output
+        }
+        else
+        {
+            throw $output
+        }
+    }
+}
+
```
Comment (ferventcoder) [inline DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:477]: This probably can't influence the exit code of the run can it?

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 [hunk 2]
```diff
@@ -419,11 +419,60 @@ function Get-ChocoInstalledPackage ($action) {
...
+function Invoke-Chocolatey
+{
+    [CmdletBinding()]
+    [Alias("IChoco")]
+    [OutputType([int])]
+    Param
+    (
+        # Param1 help description
+        [Parameter(Position=0)]
+        $pParams
```
Comment (ferventcoder) [inline DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:450]: Can we just call this arguments?

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 [hunk 3]
```diff
@@ -419,11 +419,60 @@ function Get-ChocoInstalledPackage ($action) {
...
+    Invoke-Chocolatey "list"
+.EXAMPLE
+    Invoke-Chocolatey "list -lo"
+.EXAMPLE
+    IChoco "list -lo"
+#>
+function Invoke-Chocolatey
+{
+    [CmdletBinding()]
+    [Alias("IChoco")]
```
Comment (ferventcoder) [inline DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:444]: Let's not add an alias

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 [hunk 4]
```diff
@@ -419,11 +419,60 @@ function Get-ChocoInstalledPackage ($action) {
...
+    [OutputType([int])]
```
Comment (ferventcoder) [inline DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:445]: How will this work with `choco list -lo -r` that pipes the text output?

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 (hunk 1 above)
Comment (esiebes) [reply DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:477]: exit code is now handled in pull request 103

File: DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1 (hunk 3 above)
Comment (esiebes) [reply DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1:444]: alias removed in pullrequest 103

**Comment by esiebes** [comment]
i

**Comment by esiebes** [comment]
please see pullrequest 103.

**Comment by ferventcoder** [comment]
Closed in favor of #103

**Pull request closed without merging**